src/ContainerBuilder.php

```diff
@@ -234,17 +248,35 @@ protected function findConfigFile($file)
         throw new LoaderException(sprintf("The config file '%s' does not exist in any of the configured paths", $file));
     }
 
-    protected function findImportedConfigFile($file, $dir)
+    /**
+     * @param $file
+     * @param $dir
+     * @param $relativeDir
+     * @return string
+     * @throws LoaderException
+     */
+    protected function findImportedConfigFile($file, $dir, $relativeDir)
     {
-        $filePath = $dir . "/" . $file;
-        if (file_exists($filePath)) {
-            return $filePath;
-        }
-        throw new LoaderException(sprintf("The import file '%s' does not exist in the directory '%s'", $file, $dir));
+        $relativeFilePath = $dir . DIRECTORY_SEPARATOR . $relativeDir . DIRECTORY_SEPARATOR . $file;
+        $rootFilePath = $dir . DIRECTORY_SEPARATOR . DIRECTORY_SEPARATOR . $file;
```

Comment:
This one has the directory separator doubled. Other than that I think we're good